### PR TITLE
Revert back to Satoshi's True Vision

### PR DESCRIPTION
**Pull request by cryptagoras**
- Restores Satoshi's True Vision
- Follows [Bitcoin Clashic](https://twitter.com/BitcoinClashic)

### CODE REVIEW

**Comment by verretor** [comment]
Concept ACK

**Comment by Daxx** [comment]
Haven't studied the commits but does it matter?
"Restores Satoshi's True Vision" <--- This seals it, for me.
Would ACK in a heartbeat.

File: doc/man/bitcoin-cli.1 [hunk 1]
```diff
@@ -1,15 +1,15 @@
-.\" DO NOT MODIFY THIS FILE!  It was generated by help2man 1.47.5.
-.TH BITCOIN-CLI "1" "November 2017" "bitcoin-cli v0.16.0.0" "User Commands"
+.\" DO NOT MODIFY THIS FILE!  It was generated by help2man 1.47.3.
+.TH BITCOIN-CLI "1" "February 2017" "bitcoin-cli v0.14.0.0" "User Commands"
```
Comment (jaybny) [inline doc/man/bitcoin-cli.1]: should it be v0.15.0.0 ?

**Comment by jaybny** [comment]
whats the objective? to go back pre Aug 1?

**Comment by deadalnix** [comment]
I was answering by mail. Can you open a separate issue for that one ?

**Pull request closed without merging**